comments! (#1)
bluegenes/2022-paper-protein-kmers: content/03.results.md

## Results

K-mer analysis methods enable similarity detection as low as a single shared k-mer between divergent genomes.

> As a result, exact matching of long nucleotide k-mers has been used for taxonomic classification and similarity detection between closely related genomes, including strain-level, species-level, and genus-level comparisons (often using k-mer lengths 51, 31, and 21, respectively (CTB NOTE: metapalette uses k=41, I think; not sure there's a good citation here)).

Reviewer: ```suggestion
As a result, exact matching of long nucleotide k-mers has been used for taxonomic classification and similarity detection between closely related genomes (genus-level to strain-level comparisons using k-mer lengths between 21-51) [@mash; @metapalette].
```

At larger evolutionary distances, accumulated nucleotide divergence limits the utility of exact nucleotide k-mer matching.
Amino acid k-mers (kaa-mers) retain the benefits of fast, alignment-free exact k-mer matching, but with increased tolerance for evolutionary divergence.
Here, we evaluate the utility of amino acid k-mers for a wide range of genomic and metagenomic applications, including sequence distance estimation and taxonomic classification.

### Amino Acid k-mers can be used to discriminate between taxa

The Genome Taxonomy Database (GTDB) provides a genome-similarity-based taxonomy for bacterial and archaeal genomes [@doi:10.1038/s41587-020-0501-8].
GTDB release `rs202` encompasses 258,407 genomes from 47,895 species.
We begin by assessing the prevalance of nucleotide and amino acid k-mers of different k-mer lengths within assemblies included in GTDB.

> To make analyses at this scale tractable, we use FracMinHash sketching to randomly subsample all available k-mers, retaining approximately (\~1/1000 nucleotide k-mers) and \~1/200 amino acid k-mers (CTB: formatting??)) [@doi:10.1101/2022.01.11.475838].

Reviewer: ```suggestion
To make analyses at this scale tractable, we use FracMinHash sketching to randomly subsample all available k-mers, retaining approximately 1/1000 nucleotide k-mers and 1/200 amino acid k-mers [@doi:10.1101/2022.01.11.475838].
```

> DNA FracMinHash sketches have been shown to accurately subsample (?? CTB: rephrase 'accurately subsample') genome datasets [@doi:10.1101/2022.01.11.475838].

Reviewer: ```suggestion
DNA FracMinHash sketches have been shown to representatively subsample genome datasets [@doi:10.1101/2022.01.11.475838].
```

For the direct protein assessment, comparison proteomes were also sketched from the available proteome.
For these sketches, k-mer containment results are equally valid in both directions, so we report the mean containment for the comparison alongside the Jaccard Index.
For the second workflow, comparison genomes were 6-frame translated to build protein kaa-mers.

> As 6-frame translation introduces erroneous kaa-mers, we report only the containment estimate relative to the anchor proteome (CTB: perhaps note that the intuition here is that for long k, only correct k-mers will match).

Reviewer: ```suggestion
As 6-frame translation introduces erroneous k~aa~-mers, we report only the containment estimate relative to the anchor proteome (CTB: perhaps note that the intuition here is that for long k, only correct k-mers will match).
```
leaving in your note as a reminder to address
